plugins/outputs/cratedb/cratedb_test.go

```diff
@@ -93,66 +92,69 @@ VALUES
 	}
 }
 
+var escapeValueTests = []struct {
```

Comment:
Please do this as a function, returning a fresh new instance of the patterns for every tests. We should do this to be _absolutely_ sure there are no side-effects between the tests by (accidentally) altering those patterns.